### PR TITLE
refacor PR merged of system test workflow

### PR DESCRIPTION
**Pull request by rcohencyberarmor**
Sorry, we do not accept changes directly against this repository. Please see
CONTRIBUTING.md for information on where and how to contribute instead.

### CODE REVIEW

File: .github/workflows/pr-created.yaml [hunk 1]
```diff
@@ -16,5 +16,5 @@ jobs:
     uses: kubescape/workflows/.github/workflows/incluster-comp-pr-created.yaml@main
     with:
       CGO_ENABLED: 0
-      GO_VERSION: "1.19"
+      GO_VERSION: "1.20"
```
Comment (dwertent) [inline .github/workflows/pr-created.yaml:19]: Is it 20?
Please update the go mod as well

**Review by dwertent — CHANGES_REQUESTED** [review]

File: .github/workflows/pr-created.yaml (hunk 1 above)
Comment (rcohencyberarmor) [reply .github/workflows/pr-created.yaml:19]: @dwertent

Comment (dwertent) [reply .github/workflows/pr-created.yaml:19]: yes, it will trigger the new WF

Comment (dwertent) [reply .github/workflows/pr-created.yaml:19]: I removed this dependency... The tests will run when with the label

Comment (dwertent) [reply .github/workflows/pr-created.yaml:19]: With the release label

**Review by dwertent — APPROVED** [review]

**Pull request merged**